### ISSUE TITLE
Angular 7 many warnings

### ISSUE THREAD

**Issue by vhaxha** [swimlane/ngx-datatable#1582]
Hi 

In angular 7, there are those warnings:

npm WARN @swimlane/ngx-datatable@13.1.0 requires a peer of @angular/common@<7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @swimlane/ngx-datatable@13.1.0 requires a peer of @angular/compiler@<7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @swimlane/ngx-datatable@13.1.0 requires a peer of @angular/core@<7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @swimlane/ngx-datatable@13.1.0 requires a peer of @angular/platform-browser@<7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @swimlane/ngx-datatable@13.1.0 requires a peer of @angular/platform-browser-dynamic@<7.0.0 but none is installed. You must install peer dependencies yourself.

kind regards

**Comment by danielmoncada** [comment]
Similar warnings myself.

```cs
warning " > @swimlane/ngx-datatable@13.1.0" has incorrect peer dependency "@angular/common@<7.0.0".
warning " > @swimlane/ngx-datatable@13.1.0" has incorrect peer dependency "@angular/compiler@<7.0.0".
warning " > @swimlane/ngx-datatable@13.1.0" has incorrect peer dependency "@angular/core@<7.0.0".
warning " > @swimlane/ngx-datatable@13.1.0" has incorrect peer dependency "@angular/platform-browser@<7.0.0".
warning " > @swimlane/ngx-datatable@13.1.0" has incorrect peer dependency "@angular/platform-browser-dynamic@<7.0.0".
```

Tested, and everything seems to work still. I believe dependency versions for these just need to be updated to support 17.0

**Comment by vhaxha** [comment]
Thank you very much for your help.


Kind regards,

Vesel Haxha

________________________________

**Comment by marjan-georgiev** [comment]
Released 14.0.0 with support for angular 7.
